core/src/test/scala/com/pingcap/tispark/datasource/BaseDataSourceTest.scala

```diff
@@ -37,12 +34,6 @@ class BaseDataSourceTest(val table: String,
     initializeTimeZone()
   }
 
-  protected def initializeTimeZone(): Unit = {
-    tidbStmt = tidbConn.createStatement()
-    // Set default time zone to GMT-7
-    tidbStmt.execute(s"SET time_zone = '$timeZoneOffset'")
```

Comment:
why remove this? This seems to be an irrelevant change.

Reply:
useless since it is already available in BaseTiSparkSuite

Reply:
Gotcha.